api/constants/constants.go

```diff
@@ -24,6 +24,7 @@ const (
 	GroupPrefix     = "symphony"
 	ManagerMetaKey  = GroupPrefix + "/managed-by"
 	InstanceMetaKey = GroupPrefix + "/instance"
+	NotFound        = "Not Found"
```

Comment:
use v1alpha2.IsNotFound instead. error type comparison is better than error context comparison.

Reply:
Not correct. Inside MVP you can always use `v1alpha2.IsNotFound` as the error type is internal. But this error is coverted from API call which is a typical genetic error, cannot be converted to v1alpha.error. Sample log: `level=error msg="V (InstanceContainers): onInstanceContainers failed - Not Found: failed to get object (caused by: instancecontainers.solution.symphony \"siteinstance\" not found)`

Reply:
String match is not reliable. Maybe in future, we need to consider if we can populate the inner error and keep the original type then do comparison.